transformers/modeling_bert.py

```diff
@@ -759,28 +838,46 @@ def tie_weights(self):
                                    self.bert.embeddings.word_embeddings)
 
     def forward(self, input_ids, attention_mask=None, token_type_ids=None, position_ids=None, head_mask=None,
-                masked_lm_labels=None):
+                masked_lm_labels=None, encoder_hidden_states=None, encoder_attention_mask=None, ltr_lm_labels=None, ):
```

Comment:
What does `ltr` means in `ltr_lm_labels`?
Maybe find a more explicit name if you don't like `lm_labels`

Reply:
(Note that `lm_labels` is fine for me)